### PR TITLE
django-nested-admin NestedStackedInline/NestedTabularInline adding new row does not forward fields

### PR DESCRIPTION
**Pull request by akshenk8**
https://github.com/yourlabs/django-autocomplete-light/issues/1099

### CODE REVIEW

File: src/dal/static/autocomplete_light/forward.js [hunk 1]
```diff
@@ -131,7 +131,7 @@
             forwardedData,
             divSelector,
             form;
-        divSelector = "div.dal-forward-conf#dal-forward-conf-for-" +
+        divSelector = "div.dal-forward-conf#dal-forward-conf-for_" +
                 element.attr("id");
```
Comment (jpic) [inline src/dal/static/autocomplete_light/forward.js]: Looks like a pretty breaking change, any chance to support both versions at once ?

Comment (akshenk8) [reply src/dal/static/autocomplete_light/forward.js]: done.

Comment (akshenk8) [reply src/dal/static/autocomplete_light/forward.js]: But can you tell me about what exactly would it be breaking ? 
As I have taken care of the id in JS, Widget and Tests.

**Comment by jpic** [comment]
Well it will work on new projects but for projects that are going to
upgrade dal before nested admin for some reason it could be nice if dal
still works if possible otherwise it doesn't matter much

**Comment by akshenk8** [comment]
@jpic does this work now or anything else has to taken care of ?

**Comment by jpic** [comment]
Well if you're going to break compatibility with older nested-admin versions, it would be great to mention it somewhere. Also, set a minimal version with >= in setup.py for django-nested-admin

**Comment by akshenk8** [comment]
I have set the minimal version number for django-nested-admin in test/requirements.txt and setup.py.
Sorry for the delays in the fix.

**Comment by jpic** [comment]
Nice, thanks !

**Pull request merged**